allel/stats/roh.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, print_function, division
+import numpy as np
+
+
+def call_roh(g, pos, ix, phet_roh=0.001, phet_nonroh=(0.0025, 0.01),
```

Comment:
Can we name this function "roh_mhmm" instead? I'm just thinking it would leave room for other functions calling roh in a different way.